braph2genesis/graphs/_MultilayerWU.gen.m

```diff
@@ -0,0 +1,256 @@
+ number between 0 and 1 and indicating the strength of the connection.
+The connectivity matrices are symmetric.
+There are connections between layers connecting the corresponding nodes.
+
+%% ¡props_update!
+
+%%% ¡prop!
+
+NAME (constant, string) is the name of the multilayer weighted undirected graph.
+%%%% ¡default!
+'MultilayerWU'
+
+%%% ¡prop!
+DESCRIPTION (constant, string) is the description of the multilayer weighted undirected graph.
+%%%% ¡default!
+'In a multilayer weighted undirected (WU) graph, layers could have different number of nodes with within-layer weighted undirected edges, associated with a real number between 0 and 1 and indicating the strength of the connection. The connectivity matrices are symmetric. There are connections between layers connecting the corresponding nodes.'
+
+%%% ¡prop!
+TEMPLATE (parameter, item) is the template of the multilayer weighted undirected graph.
+%%% ¡_prop!
+% % % TEMPLATE (parameter, item) is the graph template to set the graph and measure parameters.
+% % % %%%% ¡_settings!
+% % % 'MultilayerWU'
+
+%%% ¡prop!
+ID (data, string) is a few-letter code for the multilayer weighted undirected graph.
+%%%% ¡default!
+'MultilayerWU ID'
+
+%%% ¡prop!
+LABEL (metadata, string) is an extended label of the multilayer weighted undirected graph.
+%%%% ¡default!
+'MultilayerWU label'
+
+%%% ¡prop!
+NOTES (metadata, string) are some specific notes about the multilayer weighted undirected graph.
+%%%% ¡default!
+'MultilayerWU notes'
+
+%%% ¡prop!
+GRAPH_TYPE (constant, scalar) returns the graph type __Graph.MULTILAYER__.
+%%%% ¡default!
+Graph.MULTILAYER
+
+%%% ¡prop!
+CONNECTIVITY_TYPE (query, smatrix) returns the connectivity type __Graph.WEIGHTED__ * ones(layernumber).
+%%%% ¡calculate!
+if isempty(varargin)
+    layernumber = 1;
+else
+    layernumber = varargin{1};
+end
+value = Graph.WEIGHTED * ones(layernumber);
+
+%%% ¡prop!
+DIRECTIONALITY_TYPE (query, smatrix) returns the directionality type __Graph.UNDIRECTED__ * ones(layernumber).
+%%%% ¡calculate!
+if isempty(varargin)
+    layernumber = 1;
+else
+    layernumber = varargin{1};
+end
+value = Graph.UNDIRECTED * ones(layernumber);
+
+%%% ¡prop!
+SELFCONNECTIVITY_TYPE (query, smatrix) returns the self-connectivity type __Graph.NONSELFCONNECTED__ on the diagonal and __Graph.SELFCONNECTED__ off diagonal.
+%%%% ¡calculate!
+if isempty(varargin)
+    layernumber = 1;
+else
+    layernumber = varargin{1};
+end
+value = Graph.SELFCONNECTED * ones(layernumber);
+value(1:layernumber+1:end) = Graph.NONSELFCONNECTED;                
+
+%%% ¡prop!
+NEGATIVITY_TYPE (query, smatrix) returns the negativity type __Graph.NONNEGATIVE__ * ones(layernumber).
+%%%% ¡calculate!
+if isempty(varargin)
+    layernumber = 1;
+else
+    layernumber = varargin{1};
+end
+value = Graph.NONNEGATIVE * ones(layernumber);
+
+%%% ¡prop!
+A (result, cell) is the cell containing the multilayer weighted adjacency matrices of the multilayer weighted undirected graph.
```

Comment:
it contains more than adjacency matrices. So either you say that it contains the multilayer weighted supra-adjacency matrix or you say that it contains the within-layer weighted adjacency matrices of the multilayer weighted undirected graph and the connections between layers.